src/commands/registries/logInToDockerCli.ts

```diff
@@ -42,10 +33,10 @@ export async function logInToDockerCli(context: IActionContext, node?: RegistryT
                     client => client.login({
                         username: username,
                         passwordStdIn: true,
-                        registry: creds.registryPath,
+                        registry: '' // TODO: change this
```

Comment:
We can change this to be the baseUrl's host now right?

Reply:
ahhh yup good catch